src/aiida/orm/nodes/process/process.py

```diff
@@ -8,6 +8,8 @@
 ###########################################################################
 """Module with `Node` sub class for processes."""
 
+from __future__ import annotations
```

Comment:
I'm not sure why is this needed
```suggestion
```

Reply:
This is a remnant for when I added the type annotations `logging.LoggerAdapater | None`. The `|` for `Union` is only built-in as of Python 3.10. For older versions you have to enable it with this statement.